SalesforceCommerceCloud/pwa-kit
# V3 Storefront Preview add siteId query parameter to shopper context calls (@W-15976653@)

# Description



The PR adds the `siteId` defined in CommerceSdkProvider to the `window.STOREFRONT_PREVIEW` global for later use in the Storefront Preview Shopper Context Calls.


# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Add siteId to StorefrontPreview in `commerce-sdk-react`

# How to Test-Drive This PR

- Start the dev server for the template-retail-react-app: `npm start` and navigate to localhost:3000.
-  Print the `window.STOREFRONT_PREVIEW` object in the browser console.
- Verify the object now includes `siteId`.

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
